<?xml version="1.0"?>

<launch>
    <node pkg="lane_detector_final" type="laneDetectorNode" name="lane_detector0" args="6" output="screen" cwd="node">
        <param name="debug_mode" type="int" value="5"/>
        <param name="ipt_offsets_file" type="str" value="../data/ipt_offsets0.dat"/>                
        <param name="map_size" type="int" value="1000"/>        
        <param name="published_topic_name" type="str" value="/lane_detector0/lanes"/>        
        <param name="subscribed_topic_name" type="str" value="/camera0/image"/>
        <param name="time_functions" type="int" value="0"/>
        <param name="training_data_file" type="str" value="../data/Samples"/>
        <param name="wait_time" type="int" value="10"/>
        <param name="warp_matrix_file" type="str" value="../data/warp_matrix0.dat"/>
    </node>
    <node pkg="lane_detector_final" type="laneDetectorNode" name="lane_detector1" args="6" output="screen" cwd="node">
        <param name="debug_mode" type="int" value="5"/>
        <param name="time_functions" type="int" value="0"/>
        <param name="train_file" type="str" value="../data/Samples"/>
        <param name="subscribed_topic_name" type="str" value="/camera1/image"/>
        <param name="published_topic_name" type="str" value="/lane_detector1/lanes"/>        
        <param name="warp_matrix_file" type="str" value="../data/warp_matrix1.dat"/>        
        <param name="ipt_offsets_file" type="str" value="../data/ipt_offsets1.dat"/>                
    </node>
</launch>
